x-pack/legacy/plugins/siem/index.ts

```diff
@@ -23,7 +24,7 @@ import {
   DEFAULT_FROM,
   DEFAULT_TO,
 } from './common/constants';
-import { signalsAlertType } from './server/lib/detection_engine/alerts/signals_alert_type';
+import { getAlertType as getSignalsAlertType } from './server/lib/detection_engine/alerts/signals_alert_type';
```

Comment:
In our imports as a style we try to keep the `as` renaming down to a minimum or not at all unless we have a definitive name clash and need to use it.

This would be:

```
import { getAlertType } from './server/lib/detection_engine/alerts/signals_alert_type';
```

If you take my advice below on our style of keeping single functions in a file the same name as the file per our style it would be the original:

```ts
import { signalsAlertType } from './server/lib/detection_engine/alerts/signals_alert_type';
```

And then usage would be the same.


As an aside I might change the name to `getSignalsAlertType` as well as the file its self if I'm noticing it is tripping people up, but for this PR right now I say let's keep it the same file name and function name it was before to keep the conflicts between myself and Devin lower.

Reply:
Sounds good, I will change it back to `signalsAlertType` to avoid merge conflicts. Fixed in commit: https://github.com/elastic/kibana/pull/46954/commits/9741654daca56c197407f1cbe56ad6445c6bc42b.